components/motor/roboclaw/roboclaw.go

```diff
@@ -41,7 +47,14 @@ func (conf *Config) Validate(path string) ([]string, error) {
 	if conf.SerialBaud <= 0 {
 		return nil, utils.NewConfigValidationFieldRequiredError(path, "serial_baud_rate")
 	}
+	if conf.Address != 0 && (conf.Address < 128 || conf.Address > 135) {
+		return nil, errors.New("serial address must be between 128 and 135")
+	}
 
+	if conf.SerialBaud != 2400 && conf.SerialBaud != 9600 && conf.SerialBaud != 19200 &&
```

Comment:
[minor] Let's clean this up a little - there's a function that @susmitaSanyal wrote int `components/movementsensor/imuwit/imu.go` that checks from a list of valid baud rates and validates if the config baud rate is one of them. This is a good candidate for a utils move, Susmita :)

Reply:
Agreed. Having baud rate validation in rdk/utils will make it generalized for us to use it for other components. Olivia, let me know if you need help moving it to utils.